src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getMissions } from './Redux/missions/missionsSlice';
+import Layout from './components/Layout';
+import Rockets from './routes/Rockets';
+import Missions from './routes/Missions';
+import Profile from './routes/Profile';
+import './styles/style.css';
+import { rocketItems } from './Redux/RocketsSlice/RocketSlice';
+
+function App() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(rocketItems());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route path="/" element={<Profile />} />
```

Comment:
- [ ] It will be great to make the rockets the default view. At the moment Profile is the default view which goes against the project requirement
![default view](https://user-images.githubusercontent.com/32361963/233597777-c82c300d-0bcf-4015-82d8-58adaf075c1b.png)